### PR TITLE
Make Andy happy again (by fixing debug mode)

### LINKED ISSUE #3440: No response for /debug command in Console
### Description

*Type*: Bug

*Summary*: No response for /debug command in Console

#### Expected behavior
I can see response for `debug - On / Off` interaction in Console

#### Actual behavior

image

### Reproduction

- Open Status
- call /debug command

### Additional Information
* Status version: nightly 253
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by jeluard**
Fixes #3440 

### Summary:

Fix broken `debug` command

### Steps to test:
- Open Status
- Go to console chat
- Send `debug On` command

status: ready

### CODE REVIEW

File: src/status_im/commands/handlers/jail.cljs [hunk 1]
```diff
@@ -23,12 +23,11 @@
         (dispatch [:set-chat-ui-props {:validation-messages markup}]))
 
       result
-      (let [command' (assoc command :handler-data returned)
-            params'  (assoc params :command command')] 
-        (dispatch [:chat-send-message/send-command chat-id params']))
+      (let [command' (assoc command :handler-data returned)]
+        (dispatch [:chat-send-message/send-command (assoc params :command command')]))
```
Comment (dmitryn) [inline src/status_im/commands/handlers/jail.cljs]: `let` can be removed here and `assoc-in` could be used instead:
`(assoc-in params [:command :handler-data] returned)`

Comment (jeluard) [reply src/status_im/commands/handlers/jail.cljs]: :+1:

**Comment by jeluard** [comment]
@dmitryn Updated

**Comment by flexsurfer** [comment]
@jeluard there are a lot of just removing unused things, could you please point and elaborate  what was the problem and how did you fix it, thanks

**Comment by jeluard** [comment]
@flexsurfer Sure! The issue was an incorrect merge of two `fx` map that removed some `dispatch-n` : https://github.com/status-im/status-react/pull/3697/files#diff-fa96eeca924db983048b435891ca98f4R307

**Review by dmitryn — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
indeed , i am happy too! Hope @jeluard you dont mind :)

**Comment by jeluard** [comment]
@asemiankevich Happy tester? Weird!

**Comment by nabgilby** [comment]
in the latest status version on the Android store [ 0.9.19 (6034)] and latest released nightlies [0.9.20 (6058]) /debug no longer works, again it seems.

What version does it work in so I can go back to it?

**Pull request merged**